deploy/hooks/post-restore-code

```diff
@@ -17,11 +17,20 @@ if [[ $CODE_DIR == */branch/* ]]; then
   make rc_branch DEPLOY_TARGET=$TARGET
   source rc_branch
   make templates
+  make printwar APACHE_BASE_PATH=main
```

Comment:
I remove, you actually need to recreate a snapshot as the post-restore-code is in the snapshot